package.json

```diff
@@ -69,7 +69,9 @@
     "build_types": "tsc -p tsconfig.json",
     "coverage_detailed": "npx --yes type-check --detail",
     "coverage": "type-coverage -p tsconfig.json --at-least 90",
-    "circular": "npx --yes dpdm --transform ./srcts/src/index.ts"
+    "circular": "npx --yes dpdm --transform ./srcts/src/index.ts",
+    "prepack": "cp README.md README-orig.md && cp npm-README.md README.md",
```

Comment:
Note: It looks like `yarn pack` does *not* ignore README-orig.md (so it gets included in the package), but `npm pack` *does* ignore the file, which is the behavior that we want.